examples/language-modeling/llama3_ds_zero3_4k.json

```
@@ -0,0 +1,25 @@
+{
+    "steps_per_print": "auto",
+    "train_batch_size": "auto",
+    "train_micro_batch_size_per_gpu": "auto",
+    "gradient_accumulation_steps": "auto",
+    "bf16": {
+        "enabled": true
+    },
+    "gradient_clipping": 1.0,
```

```suggestion
    "gradient_clipping": auto,
```

@bhargaveede Is there a specific reason you set it to 1.0 instead of auto? I noticed we've used 1.0 in other settings as well, but I'm curious to understand the rationale behind this choice.

---

@yafshar No specific reason.